docs/source/config_file.rst

```diff
@@ -191,6 +191,22 @@ section of the command line docs.
 
     This option may only be set in the global section (``[mypy]``).
 
+.. confval:: packages
+
+    :type: comma-separated list of strings
+
+    A comma-separated list of packages which should be checked by mypy if none are given on the command
+    line. This option may only be set in the global section (``[mypy]``).  This option may be used
```

Comment:
package isn't interchangeable with modules, right? Only packages will recursively type check submodules. E.g., maybe use the language from the command line docs?

Reply:
You're correct, I've copied the language from the command-line docs.